biztalk/install-and-config-guides/appendix-a-silent-installation.md

```diff
@@ -38,13 +38,35 @@ This topic lists the steps to create a silent installation of [!INCLUDE[btsBizTa
 |                                                                                                                                                                                                                                      /NOCEIP                                                                                                                                                                                                                                      |                                                |                                                                                                                                                                                                             **Applicable for BizTalk Server 2020 and newer.**<br />Disables Customer Experience Improvement Program (CEIP)<br />CEIP is enabled by default. This option is ignored by Evaluation and Developer editions setup.                                                                                                                                                                                                                                                                           |
 |                                                                                                                                                                                                                                      /InstallUpdates                                                                                                                                                                                                                                      |                                                |                                                                                                                                                                                                             **Applicable for BizTalk Server 2020 and newer.**<br />Installs most recent BizTalk Server Cumulative Update.                                                                                         |
 
+**ADDLOCAL command**
+
+The following table lists the feature wise values for /ADDLOCAL command for BizTalk Server 2020 setup.
+- All features: /ADDLOCAL ALL 
```

Comment:
Will the readability improve if the feature name is in bold:
- **All features**: /ADDLOCAL ALL

or, putting second part in code formatting:
- All features: `/ADDLOCAL ALL`

or both